src/content/subgames.cpp

```diff
@@ -77,8 +77,16 @@ struct GameFindPath
 
 std::string getSubgamePathEnv()
 {
+	static bool has_warned = false;
 	char *subgame_path = getenv("MINETEST_SUBGAME_PATH");
-	return subgame_path ? std::string(subgame_path) : "";
+	if (subgame_path && !has_warned) {
+		warningstream << "MINETEST_SUBGAME_PATH is deprecated, use MINETEST_GAME_PATH instead."
+				<< std::endl;
+		has_warned = true;
+	}
+
+	char *game_path = getenv("MINETEST_GAME_PATH");
+	return game_path ? std::string(game_path) : subgame_path ? std::string(subgame_path) : "";
```

Comment:
Nested ternary operators are awful to look at. This is more verbose but also easier to read: (not tested)
```suggestion
	if (game_path)
		return std::string(game_path);
	if (subgame_path)
		return std::string(subgame_path);
	return "";
```